dataRetriever/interface.go

```diff
@@ -91,8 +91,10 @@ const (
 	TrieEpochRootHashUnit UnitType = 17
 	// ESDTSuppliesUnit is the ESDT supplies storage unit identifier
 	ESDTSuppliesUnit UnitType = 18
-	// RoundNonceUnit is the round nonce storage unit identifier
-	RoundNonceUnit UnitType = 19
+	// MetaHdrRoundHashDataUnit is the meta header round-hash storage data pair unit identifier
+	MetaHdrRoundHashDataUnit UnitType = 49
+	// ShardHdrRoundHashDataUnit is the shard round-hash storage data unit identifier
+	ShardHdrRoundHashDataUnit UnitType = 50
```

Comment:
let's put this on the 200 value (or even better ~400 as to allow the creation of 256 shards) and place it after the ShardHdrNonceHashDataUnit definition. MetaHdrRoundHashDataUnit can be 19. Adjust the comment below to include the ShardHdrNonceHashDataUnit.